charts/auth-layer-proxy/templates/deployment.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "auth-layer-proxy.fullname" . }}
+  labels:
+    {{- include "auth-layer-proxy.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "auth-layer-proxy.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "auth-layer-proxy.selectorLabels" . | nindent 8 }}
+    spec:
```

Comment:
nit: please order items under spec alphabetically so it's easier to find a section

Reply:
Same comment in other sections of the yaml files where it makes sense

Reply:
✅  Done.

I've also ordered the `values.yaml`